From 0ad2dfec90c4b1f6f24ab7bfbd6ddb60ab6d3840 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 28 Nov 2017 17:55:42 +0800
Subject: [PATCH 2904/5242] MLK-17014-3 ARM64: dts: imx8mq: move hdmi 'port@0'
 node to dtsi file

commit  3f5dbfa9747175e125aeb3cef807f9db95f18858 from
https://source.codeaurora.org/external/imx/linux-imx.git

It is better to put the hdmi 'port@0' node definition
to 'fsl-imx8mq.dtsi' file.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    7 -------
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi    |    7 +++++++
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 3f09af1..40641c0 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -677,13 +677,6 @@
 
 &hdmi {
 	status = "okay";
-
-	port@0 {
-		reg = <0>;
-			hdmi_disp: endpoint {
-			remote-endpoint = <&dcss_disp0_hdmi>;
-		};
-	};
 };
 
 &csi1_bridge {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index e8ae7c5..9c3b94a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -576,6 +576,13 @@
 				<0x0 0x32e40000 0x0 0x40000>;   /* HDP SEC register */
 		interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>;
 		status = "disabled";
+
+		port@0 {
+			reg = <0>;
+			hdmi_disp: endpoint {
+				remote-endpoint = <&dcss_disp0_hdmi>;
+			};
+		};
 	};
 
 	lcdif: lcdif@30320000 {
-- 
1.7.9.5

